solc/CommandLineParser.cpp

```diff
@@ -323,6 +323,16 @@ OptimiserSettings CommandLineOptions::optimiserSettings() const
 	return settings;
 }
 
+bool CommandLineParser::parse(int _argc, char const* const* _argv, bool interactiveTerminal)
```

Comment:
```suggestion
bool CommandLineParser::parse(int _argc, char const* const* _argv, bool _interactiveTerminal)
```

Reply:
Good catch. Looks like it's been that way since I extracted this from `CommandLineInterface`.